app/renderer/components/bookmarks/bookmarksToolbar.js

```diff
@@ -86,8 +86,9 @@ class BookmarksToolbar extends ImmutableComponent {
       }
     }
     if (e.dataTransfer.files.length > 0) {
```

Comment:
no, we specifically don't want to do that because we only want these items when there are files